lib/file_utils_ext.rb

```diff
@@ -0,0 +1,7 @@
+require 'fileutils'
+
+module FileUtils
+  def self.readable_by_all?(path)
```

Comment:
Why did you go for a monkey-patch over just putting a `#readable_by_all?` on the object that needs this logic?